Random build failure (parallel build issue?)

While building the latest snapshot in Fedora, I got a build failure in Fedora 39, while in F38 and Rawhide (F40) the build was fine. The build failure in F39 seems intermittent, so I suppose it's a build parallelization issue.
The error in logs is:
```
[  8%] Building CXX object src/celimage/CMakeFiles/celimage.dir/bmp.cpp.o
cd /builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/redhat-linux-build/src/celimage && /usr/bin/g++ -DCELX -DCONFIG_DATA_DIR=\"/usr/share/celestia\" -DEIGEN_NO_DEBUG -DENABLE_NLS -DFMT_SHARED -DGIT_COMMIT=\"6899839\" -DHIP_DATA_DIR=\"/usr/share/celestia\" -DLOCALEDIR=\"/usr/share/locale\" -DNO_DEBUG -DUSE_FFMPEG -DUSE_LIBAVIF -DUSE_MINIAUDIO -DVERSION=\"1.7.0\" -D_USE_MATH_DEFINES -I/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/redhat-linux-build/src/celimage -I/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celimage -I/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src -I/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/redhat-linux-build -I/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/thirdparty/miniaudio -isystem /usr/include/ffmpeg -isystem /usr/include/eigen3 -isystem /usr/include/luajit-2.1 -isystem /usr/include/freetype2 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64   -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -DNDEBUG -std=gnu++17 -fPIC -fno-exceptions -MD -MT src/celimage/CMakeFiles/celimage.dir/bmp.cpp.o -MF CMakeFiles/celimage.dir/bmp.cpp.o.d -o CMakeFiles/celimage.dir/bmp.cpp.o -c /builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celimage/bmp.cpp
In file included from /builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celestia/hud.h:36,
                 from /builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celscript/lua/celx_celestia.cpp:22:
/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celutil/formatnum.h: In member function ‘constexpr const char* fmt::v10::formatter<celestia::util::FormattedFloat<T> >::parse(const fmt::v10::format_parse_context&) const’:
/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celutil/formatnum.h:125:13: error: there are no arguments to ‘throw_format_error’ that depend on a template parameter, so a declaration of ‘throw_format_error’ must be available [-fpermissive]
  125 |             throw_format_error("invalid format");
      |             ^~~~~~~~~~~~~~~~~~
/builddir/build/BUILD/Celestia-6899839d47a08e07a887e6e8268c2e16490a9571/src/celutil/formatnum.h:125:13: note: (if you use ‘-fpermissive’, G++ will accept your code, but allowing the use of an undeclared name is deprecated)
```

--- a/src/celutil/formatnum.cpp
+++ b/src/celutil/formatnum.cpp
@@ -65,7 +65,7 @@ struct fmt::formatter<ExtendedSubstring>
         auto it = ctx.begin();
         if (it != ctx.end() && *it != '}')
         {
-#if FMT_VERSION >= 100000
+#if FMT_VERSION >= 100100
             throw_format_error("invalid format");
 #else
             assert(0);

--- a/src/celutil/formatnum.h
+++ b/src/celutil/formatnum.h
@@ -121,7 +121,7 @@ struct fmt::formatter<celestia::util::FormattedFloat<T>>
         auto it = ctx.begin();
         if (it != ctx.end() && *it != '}')
         {
-#if FMT_VERSION >= 100000
+#if FMT_VERSION >= 100100
             throw_format_error("invalid format");
 #else
             assert(0);

